web/forms.py

```
@@ -1677,3 +1680,42 @@ class Meta:
+
+    pdf_file = forms.FileField(
+        validators=[FileExtensionValidator(allowed_extensions=["pdf"])],
+        help_text="Upload your PDF document (max 10MB)",
+        widget=forms.FileInput(attrs={"accept": "application/pdf"}),
+    )
+
+    class Meta:
+        model = PDFSubmission
+        fields = ["title", "pdf_type", "subject", "assignment", "due_date", "description", "pdf_file"]
+
+    def clean_pdf_file(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `clean_pdf_file`

(ANN202)